diff -urN a/usr.bin/ncal/Makefile b/usr.bin/ncal/Makefile
--- a/usr.bin/ncal/Makefile	2017-12-29 10:02:09.000000000 +0100
+++ b/usr.bin/ncal/Makefile	2017-12-31 11:36:22.859087652 +0100
@@ -7,6 +7,6 @@
 topdir=../..
 include $(topdir)/config.mk
 
-install-2:
-	(cd $(bindir); ln -sf ncal cal)
-	(cd $(mandir); ln -sf ncal.1 cal.1)
+#install-2:
+#	(cd $(bindir); ln -sf ncal cal)
+#	(cd $(mandir); ln -sf ncal.1 cal.1)
