diff -urN a/config.mk b/config.mk
--- a/config.mk	2014-10-17 12:54:06.000000000 +0200
+++ b/config.mk	2016-03-08 20:45:30.801893774 +0100
@@ -43,8 +43,8 @@
 
 # normal installation rule
 install-1: $(PROG)
-	install -o root -g root -m 755 $(PROG) $(bindir)
-	install -o root -g root -m 644 $(MAN) $(mandir)
+	install -m 755 $(PROG) $(bindir)
+	install -m 644 $(MAN) $(mandir)
 
 install: install-1 install-2
 
