diff -urN a/configure b/configure
--- a/configure	2004-02-05 10:34:07.000000000 +0100
+++ b/configure	2015-04-10 19:01:54.429833662 +0200
@@ -19099,7 +19099,7 @@
     case "$optimize" in
 	-O|"-O "*)
 	    optimize="-O"
-	    optimize="$optimize -fforce-mem"
+	    #optimize="$optimize -fforce-mem"
 	    optimize="$optimize -fforce-addr"
 	    : #x optimize="$optimize -finline-functions"
 	    : #- optimize="$optimize -fstrength-reduce"
